guides/v2.3/comp-mgr/install-extensions/inventory-management-installation.md

```diff
@@ -0,0 +1,115 @@
+---
+group: software-update-guide
+title: Manage Inventory Management modules
+---
+
+Inventory Management modules provide all inventory features and options for Single and Multi Source merchants to manage product quantities and stock for sales channels. These features are available in Magento 2.3.X Open Source and Commerce.
+
+These features and extensions were developed as part of the [Multi Source Inventory (MSI) project](https://github.com/magento-engcom/msi) through the Magento Community Engineering program.
+
+## Installation
+
+Inventory Management installs like a Core Bundled Extension (CBE) with all features enabled by default. No additional steps are required for installing and enabling these inventory features. 
+
+When Magento 2.3.X installs, it adds and enables the following modules in composer.json:
+
+``` json
+        'Magento_Inventory' => 1,
+        'Magento_InventoryAdminUi' => 1,
+        'Magento_InventoryApi' => 1,
+        'Magento_InventoryBundleProduct' => 1,
+        'Magento_InventoryBundleProductAdminUi' => 1,
+        'Magento_InventoryCatalog' => 1,
+        'Magento_InventorySales' => 1,
+        'Magento_InventoryCatalogAdminUi' => 1,
+        'Magento_InventoryCatalogApi' => 1,
+        'Magento_InventoryCatalogSearch' => 1,
+        'Magento_InventoryConfigurableProduct' => 1,
+        'Magento_InventoryConfigurableProductAdminUi' => 1,
+        'Magento_InventoryConfigurableProductIndexer' => 1,
+        'Magento_InventoryConfiguration' => 1,
+        'Magento_InventoryConfigurationApi' => 1,
+        'Magento_InventoryGroupedProduct' => 1,
+        'Magento_InventoryGroupedProductAdminUi' => 1,
+        'Magento_InventoryGroupedProductIndexer' => 1,
+        'Magento_InventoryImportExport' => 1,
+        'Magento_InventoryIndexer' => 1,
+        'Magento_InventoryLowQuantityNotification' => 1,
+        'Magento_InventoryLowQuantityNotificationAdminUi' => 1,
+        'Magento_InventoryLowQuantityNotificationApi' => 1,
+        'Magento_InventoryMultiDimensionalIndexerApi' => 1,
+        'Magento_InventoryProductAlert' => 1,
+        'Magento_InventoryReservations' => 1,
+        'Magento_InventoryReservationsApi' => 1,
+        'Magento_InventoryCache' => 1,
+        'Magento_InventorySalesAdminUi' => 1,
+        'Magento_InventorySalesApi' => 1,
+        'Magento_InventorySalesFrontendUi' => 1,
+        'Magento_InventoryShipping' => 1,
+        'Magento_InventorySourceDeductionApi' => 1,
+        'Magento_InventorySourceSelection' => 1,
+        'Magento_InventorySourceSelectionApi' => 1,
+        'Magento_InventoryShippingAdminUi' => 1,
+```
+
+## Enable features
+
+All Inventory Management features enable by default. The **Manage Stock** option in the Admin enables and disables tracking of inventory quantities, calculations of aggregated salable quantities, and reservation management for tracking purchases through to invoice and shipment. 
```

Comment:
I think you probably need to rewrite this sentence so that "tracking" isn't used twice. The construction is a bit stilted, and I don't know how to fix it.

Reply:
Totally revised to shift the focus!